stemcell_builder/stages/system_alicloud/apply.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+if [[ "${DISTRIB_CODENAME}" != 'xenial' ]]; then
+  echo "acpiphp" >> $chroot/etc/modules
```

Comment:
Why is this module being loaded?

Reply:
This change was introduced with commit https://github.com/cloudfoundry/bosh-linux-stemcell-builder/commit/cb71dbaab858085cb9fc9c71dcbde95f8ebe18fa.
I guess it was related to ubuntu trusty. On bionic it is loaded by default
```
> grep "CONFIG_HOTPLUG_PCI_ACPI=" /boot/config-`uname -r`
CONFIG_HOTPLUG_PCI_ACPI=y
```
Interestingly the code snippet does also exist for aws and openstack:
* https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-bionic/master/stemcell_builder/stages/system_aws_modules/apply.sh#L8-L10
* https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-bionic/master/stemcell_builder/stages/system_openstack_modules/apply.sh#L8-L10

I think it can be removed. What is your opinion?

Reply:
It looks like this module was introduced to fix an issue in Ubuntu Trusty. So we can remove it from xenial and bionic. We will merge PR and clean up this.